core/server/apps/amp/lib/router.js

```diff
@@ -12,7 +14,7 @@ var path                = require('path'),
 function controller(req, res, next) {
     var defaultView = path.resolve(__dirname, 'views', 'amp.hbs'),
         paths = templates.getActiveThemePaths(req.app.get('activeTheme')),
-        data = req.amp;
+        data = req.body || {};
```

Comment:
Not sure, if you should use req.amp here, as this was removed on the LTS branch with the equivalent AMP PR. If we still want to use an amp property, I'll revert this change. @kirrg001 ?